### PR TITLE
Master kanban fix seb

### PR DESCRIPTION
**Pull request by seb-odoo**
Part of task 34045

Use rebase-ff

### CODE REVIEW

**Comment by seb-odoo** [comment]
@qsm-odoo small PR to merge. Already OK'ed by framework team.

File: addons/web/static/src/js/views/basic/basic_model.js [hunk 1]
```diff
@@ -4139,7 +4139,7 @@ var BasicModel = AbstractModel.extend({
                     }
                 } else {
                     dataPoint = self._makeDataPoint({
-                        context: list.context,
+                        context: list.getContext(),
```
Comment (qsm-odoo) [inline addons/web/static/src/js/views/basic/basic_model.js]: It's only a master fix ? Not for 12.0 at least ?

File: addons/web/static/src/js/views/kanban/kanban_record.js [hunk 2]
```diff
@@ -183,10 +183,10 @@ var KanbanRecord = Widget.extend({
     _getImageURL: function (model, field, id, cache, options) {
         options = options || {};
         var url;
-        if (this.record[field] && this.record[field].value && !utils.is_bin_size(this.record[field].value)) {
+        if (this.record[field] && this.record[field].raw_value && !utils.is_bin_size(this.record[field].raw_value)) {
```
Comment (qsm-odoo) [inline addons/web/static/src/js/views/kanban/kanban_record.js]: Same for this commit: only master fix ?

File: addons/web/static/src/js/views/basic/basic_model.js (hunk 1 above)
Comment (aab-odoo) [reply addons/web/static/src/js/views/basic/basic_model.js]: i wouldn't risk this in 12 ^^

Comment (qsm-odoo) [reply addons/web/static/src/js/views/basic/basic_model.js]: There is no risk, we got your tests ! :p Anyway, your call

**Comment by seb-odoo** [comment]
So master it is? I will need to rebase my other PR on this before merging it.

**Comment by qsm-odoo** [comment]
@robodoo delegate+
Let's trust @aab-odoo opinion, you can merge in master to move on with this :)

**Comment by seb-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by qsm-odoo** [comment]
@robodoo rebase-ff

**Comment by JKE-be** [comment]
@aab-odoo

Can you check this fix plz and introduce another one.
It introduces a bug in saas-12.2

Eg: go to contact, and you will see that the company picture is not correct on res.partner.

Thks very mouchhh !!!

**Comment by seb-odoo** [comment]
@aab-odoo I think the problem is that for the problematic case on `_getImageURL` the given `id` parameter is different than `this.id`, which is never verified.

**Pull request closed without merging**